src/cmd_kv.cc

```diff
@@ -48,4 +50,78 @@ void SetCmd::DoCmd(PClient* client) {
   client->SetRes(CmdRes::kOk);
 }
 
-}  // namespace pikiwidb
\ No newline at end of file
+AppendCmd::AppendCmd(const std::string& name, int16_t arity)
+    : BaseCmd(name, arity, CmdFlagsWrite, AclCategoryWrite | AclCategoryString) {}
+
+bool AppendCmd::DoInitial(PClient *client) {
+  client->SetKey(client->argv_[1]);
+  return true;
+}
+
+void AppendCmd::DoCmd(PClient *client) {
+  PObject* value;
+  PError err = PSTORE.GetValueByType(client->Key(),value, PType_string);
```

Comment:
这里注意下编码风格，逗号后加一个空格。71行，99行同理
```suggestion
  PError err = PSTORE.GetValueByType(client->Key(), value, PType_string);
```